migration/src/app.ts

```diff
@@ -9,7 +9,7 @@ import MigrationService from "./migrate/service";
 export default class App {
   private logger = console;
   private pg = new Postgres("default");
-  private fb = new Firebase(config.get("firebase"));
+  private fb = new Firebase();
```

Comment:
should still be passing it as param to decouple from the firebase module

Reply:
is this for the case that we want to pass in more options? how do you think we should create the firebase options object, that's what i wanted to avoid doing.